cylc/flow/network/authorisation.py

```diff
@@ -40,17 +38,7 @@ def wrapper(fcn):
         def _call(self, *args, user='?', meta=None, **kwargs):
             if not meta:
                 meta = {}
-            host = meta.get('host', '?')
-            prog = meta.get('prog', '?')
-            comms_method = meta.get('comms_method', '?')
-
-            # Hardcoded, for new - but much of this functionality can be
-            # removed more swingingly.
-            LOG.debug(
```

Comment:
@oliver-sanders, just thinking about removing this line. It does remove the `comms_method` which may be useful for debugging in future? Want me to put any logging information about the comms method to replace this, or are you happy with the info loss from the removal?

Reply:
We can pass the comms_method down with the metadata.